### ISSUE TITLE
[Question] References, links, and summaries

### ISSUE THREAD

**Issue by darrensapalo** [solana-foundation/core-community-call#11]
Hi there. I am currently studying more about Solana, Rust, and Anchor - so I am still in the early stages of becoming a dApp developer. Even if I am new, I do want to surround myself around the Solana core team so that I can understand how and what the Solana team is doing, understand the organizational design around the technology I am studying, and understand Solana as a technology and as a community in general.

So I was listening to [Community Call 1](https://github.com/solana-foundation/core-community-call/blob/main/call-notes/call_1.md). I observed that it was mostly organizational design (how do we make decisions, how do we want to work with each other, what should we be including in community calls). It was also very fascinating that you guys are sharing observations from the Ethereum community as well, so I was learning a lot.

As a newbie, I was encountering a few new terminologies that were unfamiliar to me:

- SIMD
- EIP
- Firedancer
- Jito
- Mango
- etc.

 I know how to follow my curiosity and how to read more about them, but I was thinking these transcript documents can have their readability improved, especially for people who are like me who are new and want to understand what the core community is talking about.

I was wondering if the team would appreciate pull requests to that add new things to these core community call transcript markdown files:

## Hyperlinks

External links that point to these concepts (e.g. Wikipedia reference to what SIMD is)

Value: For newbies like me who are interested to understand more about what is being discussed, external links will improve accessibility.

Additionally, a Wiki can be set up here in the core community calls to include an ELI5 that explains those concepts, or introduces those organizations.

# Open discussions

A new section maybe called "open discussions" that enumerates key items that did not result to a decision, resolution, conclusion. 

Value: This might be a useful summary to review for preparation on the succeeding core community calls. If the topic isn't intended for the core community calls, then this is a good way to make sure open discussions don't get lost in transcripts. The summary will provide a way to ensure we don't lose out on conversations that may not be as urgent or important to be handled in the core community calls.

> Jacob Creech: And then a question for the specs: whenever we have a SIMD proposed and it's accepted, do you all want to wait until after like it's actually implemented across the different clients or do you, like when do the specs change in accordance to like upgrades the network?
> 
> Richie Patel: I mean as far as I can tell, SIMDs are proposed changes and specs are authoritative sources of what the protocol actually is and merging that before and anything is activated on mainnet is I guess technically wrong, but I don't know if we want to split hairs here. 

The above is an example of a topic that was lightly stumbled upon but was regarded with lower priority. The question of "when should proposed spec changes get accepted in relation to network upgrades?" seems important but not urgent, at least within the context of the call.

---

Thanks for your time.

This may or may not be the right place to do this, but for sure- I want to stick around in your core community conversations, understand what you guys are talking about, and leave notes for myself and for others so we can eventually participate. For now, I think this is what I can contribute that I know would be beneficial to curious learners like me.

Please let me know if you think there would be a better way to contribute.

**Comment by jacobcreech** [comment]
I appreciate all the feedback. Thankfully this repository is open to anyone to contribute! I will note though on things like hyperlinks and terminology - this may be more suited for someone to create a website to host core community call notes + SIMD information. Anyone in the community is welcome to create such a website and it could help a ton of new people looking to contribute in the future.
